src/redux/books/books.js

```diff
@@ -0,0 +1,33 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+const initialState = {
+  books: [],
+};
+
+export const bookReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return {
+        ...state,
+        books: [...state.books, action.payload],
+      };
+    case REMOVE_BOOK:
+      return {
+        ...state,
+        books: state.books.filter((book) => book.id !== action.payload),
+      };
+    default:
+      return state;
+  }
+};
+
+export const addBook = (book) => ({
+  type: ADD_BOOK,
+  payload: book,
+});
+
+export const deleteBook = (id) => ({
+  type: REMOVE_BOOK,
+  payload: id,
+});
```

Comment:
You created and exported **Action Creators**. (`addBook` and `removeBook` functions) ✔

Reply:
Yeah but what about that? I dont understand what youre trying to say, should I remove it?

Reply:
Just pointing out the things you did well 😉